plugins/woocommerce-blocks/assets/js/base/components/country-input/country-input.tsx

```diff
@@ -20,7 +20,6 @@ export const CountryInput = ( {
 	label,
```

Comment:
@samueljseay this component has a `style.scss` which imports `@import "node_modules/wordpress-components/src/combobox-control/style";`. We should also remove that

Reply:
@ralucaStan I think things like that should be done in a follow up, I left them in because they might have impacts on styling since we reuse so many of the same classes to ensure everything looks and behaves the same. I think it would be safest to do this kind of removal/clean up as a follow up PR.

What do you think?

Reply:
> @ralucaStan I left them in because they might have impacts on styling since we reuse so many of the same classes to ensure everything looks and behaves the same. I think it would be safest to do this kind of removal/clean up as a follow up PR.
> 
> What do you think?

Makes sense. so all good from my side as long as you create an issue and pick it up soon

Reply:
@ralucaStan yep good point. I've created an issue https://github.com/woocommerce/woocommerce/issues/47210